app/geo/gui/src/ts/twxplore/gui/geo/components/map/MapLayout.tsx

```diff
@@ -175,6 +176,21 @@ const MapLayoutImpl: React.FunctionComponent = () => {
 
             <SelectionPanel />
           </Grid>
+
+          <Grid
+            container
+            item
+            spacing={0}
+            direction="column"
+            className={classes.drawerGridContainer}
+            justify={"flex-start"}
+          >
+            <Grid>
```

Comment:
Grid item

Reply:
This is a grid item (and container). It's both. Which seems to be fine by material-ui standards.

Reply:
My comment was on line 188, the inner <Grid>.

Reply:
Oh, I see it now